man/zram-generator.conf.md

```diff
@@ -71,6 +71,13 @@ Devices with the final size of *0* will be discarded.
 
   If unset, none will be configured and the kernel's default will be used.
 
+* `writeback-dev`=
+
+  Write incompressible pages, for which no gain was achieved, to the specified block device under memory pressure.
```

Comment:
Maybe "writeback-device" ? We don't use abbreviations in other option names?

I think some more explanation what the valid values are is needed?
```
"Takes a path to a block device, e.g. `/dev/disk/by-partuuid/22ecf232-c8a3-4627-b726-72f84d43de82` or `/dev/mapper/some-device/`."
```
?

Reply:
"writeback-device" definitely, I was expecting many more options to use it as a prefix from the kernel documentation, but it turned out that was a lie, and didn't elongate it back.

Added the blurb, too.